script.js

```diff
@@ -81,7 +64,7 @@ function addGenerateCompareUrlButton() {
 		}
 		
 		firstSelectedCommitSHA = selectedCheckboxes[0].value;
-		secondSelectedCommitSHA = checkboxes[lastIndexOfSelectedCheckboxes + 1].value;
+		secondSelectedCommitSHA = selectedCheckboxes[1].value;
```

Comment:
Ah OK, I didn't realize this was the expected behavior.

I see two problems of this though:
- someone that is a heavy user of github's compare function might think that it is confusing to relearn a new behavior
- if you select the last checkbox of the page I suspect you will try to reach for a checkbox that doesn't exist which would result in a js error